### ISSUE TITLE
Allow priority configuration in PMD formatter

### ISSUE THREAD

**Issue by yrtimiD** [palantir/tslint#1420]
Feature proposal:
When using PMD format all warnings are reported with priority=1 (guess it's a high).
Because tslint doesn't supports different priorities, I want to have separate tslint.json files with different rule sets and want to report their results differently to PMD files.
So, if tslint will support defining of PMD priority I'll be able to do something like next:
```
tslint --config=tslint_warn.json --out=pmd_warn.xml --pmd-priority=2
tslint --config=tslint_err.json --out=pmd_err.xml --pmd-priority=1
```

**Comment by JKillian** [comment]
Hi @yrtimiD, I understand why you'd like this feature, but it doesn't make sense for us to add a special flag specifically for the PMD Formatter to our API. However, adding a way to set the priority of rules is definitely something we'd like to do (see #345) and I believe it's the right way to proceed.
